js/electron/transcoding.js

```diff
@@ -307,13 +438,38 @@ function transcodingProcessor(electronIpcMain) {
       try {
         buffer = await promise;
       } catch (err) {
-        if (err === 'TRANSCODE_ABORT') {
-          const err = Error('TRANSCODE_ABORT');
-          e.sender.send('transcode-video-error', err);
-          return;
+        switch (err) {
```

Comment:
### More errors handled here
- **Transcode abort** - happens on user canceled transcoding.
- **ErrorFileAccess** - happens on error with accessing video.
- **FFBinNotDownloaded** - happens when FF Binaries download error occurred.
- **VerticalVideo** - happens when video is vertical
- **UnhandledError** - happens on some unhandled error. Logging it with `console.error` for debugging.